util/options.py

```diff
@@ -258,26 +261,35 @@ def try_read_experiments_from_images(
         for arg in args:
             # Don't expand wildcards if URI-style filename
             if "*" in arg and not get_url_scheme(arg):
-                args_new.extend(glob(arg))
+                filenames = glob(arg)
+                if not filenames:
+                    logger.warning("No files found with pattern %s" % arg)
+                else:
+                    args_new.extend(glob(arg))
```

Comment:
No need to perform the `glob()` twice. Might be preferable to switch the order of the if/else statements to avoid the need for `if not`:
```suggestion
                if filenames:
                    args_new.extend(filenames)
                else:
                    logger.warning(f"No files found with pattern {arg}")
```

Reply:
Removed the `glob` call. 
I thought logging shouldn't should f strings - is this no longer the case?

> [internally the logging package uses %-formatting to merge the format string and the variable arguments](https://docs.python.org/3/howto/logging-cookbook.html#formatting-styles)